packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -24,61 +24,61 @@
 						<span slot="title" class="item-title-root">{{ $locale.baseText('mainSidebar.workflows') }}</span>
 					</template>
 
-					<n8n-menu-item index="workflow-new">
+					<n8n-menu-item index="workflow-new" style="padding-left: 30px;">
```

Comment:
Reminder to update this. Done as it is for Design Review, how it will look like.